example/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ add_library(minimal_publisher SHARED
   src/minimal_publisher.cpp
 )
 target_include_directories(minimal_publisher PUBLIC
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include/generate_parameter_library_example>
```

Comment:
Does this example actually get installed or is this just an example of how someone would install a project?

Reply:
It is just an example project but I would like it to be as correct as possible in case someone copies from it.